src/pages/projectSettings/tabs/NotificationsTab/getFormSchema.tsx

```diff
@@ -41,6 +44,53 @@ export const getFormSchema = (
             },
           },
         },
+        ...(projectType !== ProjectType.Repo && {
+          banner: {
+            type: "object" as "object",
+            title: "",
```

Comment:
I just have a suggestion to maybe put the title outside of the card, otherwise it sort of looks like it falls under the Performance Plugins section

<img width="802" alt="Screenshot 2023-05-02 at 4 55 12 PM" src="https://user-images.githubusercontent.com/47064971/235784478-e6f97ae7-96e9-46eb-ad27-b06768c558b5.png">